apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -65,11 +82,36 @@ export const StatusBar = ({
           </span>
         </div>
       </div>
+      {/*center*/}
+      <div className="w-full flex justify-center items-center"></div>
       {/*right side*/}
-      <div className="w-full flex space-x-1 justify-end">
+      <div className="w-full flex gap-2 items-center justify-end">
         <AutoUpdater />
         <DiscordButton />
         {showThemeToggle && <ThemeToggle />}
+
+        <Modal visible={showRestartModal}>
+          <div className="flex flex-col gap-4 max-w-lg items-center">
+            <h3
+              className={classNames(
+                theme.text1,
+                "text-base font-semibold leading-6",
+              )}
+            >
+              Reset Application
+            </h3>
+            <p className={classNames(theme.text2, "text-sm text-center")}>
+              Are you sure you want to reset all state and restart the
+              application?
+            </p>
+            <div className="flex justify-around gap-2">
+              <Button onClick={() => setShowRestartModal(false)}>Cancel</Button>{" "}
+              <Button onClick={restart} dataTestid="restart-simulator-button">
+                Reset
+              </Button>
+            </div>
+          </div>
+        </Modal>
```

Comment:
Better to move it out of the flex container. Even though it uses portals behind the hood, I think the flex gap may apply when the modal is visible.

Reply:
done!